src/exercise_ui/client/i18n.cljs

```diff
@@ -18,4 +18,4 @@
   (reset! language lang))
 
 (defn value [i18n-map]
-  (get i18n-map @language i18n-map))
+  (get i18n-map @language (:en-US i18n-map)))
```

Comment:
Also thought to add something more robust to try to avoid errors at all, like 
```clojure
(or (:en-US i18n-map)
      "missing translation")
```

But I prefer to throw an error and we know that is missing, or we have an issue with the translation structure itself so we can fix, rather than show a default text and maybe have a confuse UX for the user.